core-library/src/main/java/org/silverpeas/core/contribution/attachment/webdav/WebdavService.java

```diff
@@ -24,6 +24,12 @@
 package org.silverpeas.core.contribution.attachment.webdav;
 
 import org.silverpeas.core.contribution.attachment.model.SimpleDocument;
+import org.silverpeas.core.contribution.attachment.webdav.impl.WebdavContentDescriptor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Optional;
 
 public interface WebdavService {
```

Comment:
The WebdavService can it be renamed in a more generic way like RemoteContentService? So by this way it can have several implementation depending on the protocol used to access remotely the document's content: WebDAV, WOPI, ... Or it can have a single implementation and several protocol bindings (one for WebDAV, another one for WOPI, ...) so by this way it should be more easily to figuring out by what remote way a document content is accessed and then to manage conflicts between different access on the same document file.